financial-connections/src/main/java/com/stripe/android/financialconnections/repository/FinancialConnectionsManifestRepository.kt

```diff
@@ -207,6 +204,7 @@ private class FinancialConnectionsManifestRepositoryImpl(
             url = consentAcquiredUrl,
             options = apiOptions,
             params = mapOf(
+                "expand" to listOf("active_auth_session"),
```

Comment:
PR title: "Synchronize API changes"

What are these "expand" additions?

description mentions this a little bit: "expands active_auth_session"